Projekty/Projekt2/PatrykTomaszewskiMateuszZiemla/Milestone 1.ipynb

```diff
@@ -0,0 +1,1380 @@
+{
```

Comment:
<p>Tutaj bardziej by się przydał wykres explained variance&nbsp;od n_components. Możliwe, że to pomoże dobrać optymalną liczbę składowych. Do przemyślenia przed KM2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>